### PR TITLE
Integrate new design

### PR DESCRIPTION
**Pull request by pyromaniackeca**

### CODE REVIEW

File: source/layout.erb [hunk 1]
```diff
@@ -3,83 +3,60 @@
...
-          <div class="row">
-            <div class="col-lg-10 col-sm-12 col-12  col-lg-offset-1">
-              <div class="panel panel-default">
+        <div class="row">
+          <div class="col-md-offset-2 col-md-8">
 
-                <%= yield  %>
+            <%= yield  %>
+
+            <%= partial(:creative_commons) %>
```
Comment (markoa) [inline source/layout.erb]: Documentation is not in CC. Please remove this and the partial.

File: source/_footer.html.erb [hunk 2]
```diff
@@ -1,10 +1,10 @@
 <footer class="page-footer">
   <div class="container">
     <div class="row">
-      <div class="col-sm-4 col-md-4 col-lg-4">
+      <div class="col-md-4">
         <section class="footer-contact">
           <div class="pull-left footer-logo">
-            <a href="/"><img alt="Semaphore" height="45" src="/docs/assets/img/design/logo-semaphore.svg" width="181"></a>
+            <a href="/community"><img alt="SemaphoreCI" height="45" src="/docs/assets/img/design/logo-semaphore.svg" width="181"></a>
```
Comment (markoa) [inline source/_footer.html.erb]: SemaphoreCI => Semaphore documentation for continuous integration and deployment

Comment (markoa) [inline source/_footer.html.erb]: s/SemaphoreCI/Semaphore

File: source/_footer.html.erb [hunk 3]
```diff
@@ -29,17 +29,19 @@
           </ul>
         </section>
       </div>
-      <div class="col-sm-2 col-md-2 col-lg-2">
+      <div class="col-md-2">
         <section>
           <h2 class="footer-section-title">Resources</h2>
           <ul class="list-unstyled">
             <li><a href="/docs">Documentation</a></li>
+            <li><a href="/community/tutorials">Tutorials</a></li>
```
Comment (markoa) [inline source/_footer.html.erb]: Please remove these two until we officially launch the community.

File: source/_footer.html.erb (hunk 2 above)
Comment (pyromaniackeca) [inline source/_footer.html.erb]: Can you please clarify?

Comment (markoa) [inline source/_footer.html.erb]: Content of the alt tag should be "Semaphore", not "SemaphoreCI". We haven't renamed the service yet.

**Pull request merged**